app/screens/global_drafts/components/draft_empty_component/index.tsx

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {useIntl} from 'react-intl';
+import {Image, Text, View} from 'react-native';
```

Comment:
Somewhere else you are using Image from expo-image for the same purpose, which makes sense, change it here

Reply:
Done.